README.rst

```diff
@@ -24,15 +24,26 @@ RNA-Seq Normalization
     :target: https://pepy.tech/project/rnanorm
     :alt: Number of downloads from PyPI
 
-Normalization of RNA-seq gene expression data. Supported methods are Transcript
-per kilobase million (TPM) and Counts per million (CPM).
+Normalization of RNA-seq gene expression data. Supported methods:
+
+* Counts per million (CPM)
+* Transcript per kilobase million (TPM)
+* Quantile normalization to average distribution
 
 The TPM normalization can either accept pre-computed gene lengths on the input
 or compute gene lengths from gene annotation in GTF format, using the union
 exon-based approach. The computed gene lengths are identical to the lengths
 reported by featureCounts (validated for *Homo sapiens*, *Mus musculus*,
 *Rattus norvegicus*, and *Macaca mulatta* of ENSEMBL and UCSC annotations).
 
+The procedure is implemented as described on Wikipedia_. First, we compute an
```

Comment:
It is not clear what this paragraph refers to. Based on the link itself, it seems to describe quantile normalization. The link label is too generic, perhaps change it so that `Quantile normalization` word links to wikipedia.

Reply:
Thanks, fixed!